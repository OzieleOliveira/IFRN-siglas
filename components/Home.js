import * as React from 'react';
import { Text, View, StyleSheet, TouchableHighlight } from 'react-native';

export default function Home(props){
  return (
    <View style={Styles.container}>

        <TouchableHighlight testID ='bt'
        style={Styles.bloco}
                onPress={() => props.navigation.navigate('Significado',{name:<b>DG</b>, sig:'Direção Geral'})}
                
      >
          <Text testID='btn'>DG</Text>
      </TouchableHighlight>

    
      <TouchableHighlight
        style={Styles.bloco} testID ='bt2'
                onPress={() => props.navigation.navigate('Significado',{name:<b>GABIN</b>, sig:'Chefia de Gabinete'})}
      >
          <Text testID='btn1'>GABIN</Text>
      </TouchableHighlight>

          <TouchableHighlight
        style={Styles.bloco}
                onPress={() => props.navigation.navigate('Significado',{name:<b>CTI</b>, sig:'Coordenação de Tecnologia da Informação'})}
      >
          <Text testID='btn2'>CTI</Text>
      </TouchableHighlight>


      <TouchableHighlight
        style={Styles.bloco}
              onPress={() => props.navigation.navigate('Significado',{name:<b>COGPE</b>, sig:'Coordenação de Gestão de Pessoas'})}
      >
          <Text testID='btn3'>COGPE</Text>         
      </TouchableHighlight>

          <TouchableHighlight
        style={Styles.bloco}
              onPress={() => props.navigation.navigate('Significado',{name:<b>COAPAC</b>,sig:'Coordenação de Apoio Acadêmico'})}
      >
          <Text testID='btn4'>COAPAC</Text>
      </TouchableHighlight>

          <TouchableHighlight
        style={Styles.bloco}
              onPress={() => props.navigation.navigate('Significado',{name:<b>COAES</b>, sig:'Coordenação de Atividades Estudantis'})}
      >
          <Text testID='btn5'>COAES</Text>
      </TouchableHighlight>

          <TouchableHighlight
        style={Styles.bloco}
              onPress={() => props.navigation.navigate('Significado',{name:<b>COEX</b>, sig:'Coordenação de Extensão'})}
      >
          <Text testID='btn6'>COEX</Text>
      </TouchableHighlight>

          <TouchableHighlight
        style={Styles.bloco}
              onPress={() => props.navigation.navigate('Significado',{name:<b>COPEIN</b>, sig:'Coordenação de Pesquisa e Inovação'})}
      >
          <Text testID='btn7'>COPEIN</Text>
      </TouchableHighlight>

          <TouchableHighlight
        style={Styles.bloco}
              onPress={() => props.navigation.navigate('Significado',{name:<b>DIAC</b>, sig:'Diretoria Acadêmica'})}
      >
          <Text testID='btn8'>DIAC</Text>
      </TouchableHighlight>

          <TouchableHighlight
        style={Styles.bloco}
               onPress={() => props.navigation.navigate('Significado',{name:<b>SEAC</b>, sig:'Secretaria Acadêmica'})}
      >
          <Text testID='btn9'>SEAC</Text>
      </TouchableHighlight>

          <TouchableHighlight
        style={Styles.bloco}
              onPress={() => props.navigation.navigate('Significado',{name:<b>COLAB</b>, sig:'Coordenação de Laboratórios'})}
      >
          <Text testID='btn10'>COLAB</Text>
      </TouchableHighlight>

          <TouchableHighlight
        style={Styles.bloco}
              onPress={() => props.navigation.navigate('Significado',{name:<b>DIAD</b>, sig:'Diretoria de Administração'})}
      >
          <Text testID='btn11'>DIAD</Text>
      </TouchableHighlight>

          <TouchableHighlight
        style={Styles.bloco}
              onPress={() => props.navigation.navigate('Significado',{name:<b>COSGEM</b>, sig:'Coordenação de Serviços Gerais e Manutenção'})}
      >
          <Text testID='btn12'>COSGEM</Text>
      </TouchableHighlight>

          <TouchableHighlight
        style={Styles.bloco}
              onPress={() => props.navigation.navigate('Significado',{name:<b>COMPAT</b>, sig:'Coordenação de Material e Patrimônio'})}
      >
          <Text testID='btn13'>COMPAT</Text>
      </TouchableHighlight>

          <TouchableHighlight
        style={Styles.bloco}
              onPress={() => props.navigation.navigate('Significado',{name:<b>COFINC</b>, sig:'Coordenação de Finanças e Contratos'})}
      >
          <Text testID='btn14'>COFINC</Text>
      </TouchableHighlight>

          <TouchableHighlight
        style={Styles.bloco}
              onPress={() => props.navigation.navigate('Significado',{name:<b>NAPNE</b>, sig:'Núcleo de Atendimento às Pessoas com Necessidades Educacionais Especiais'})}
      >
          <Text testID='btn15'>NAPNE</Text>
      </TouchableHighlight>

          <TouchableHighlight
        style={Styles.bloco}
              onPress={() => props.navigation.navigate('Significado',{name:<b>NEABI</b>, sig:'Núcleo de Estudos Afro-brasileiros e Indígenas do Campus Canguaretama'})}
      >
          <Text testID='btn16'>NEABI</Text>
      </TouchableHighlight>

          <TouchableHighlight
        style={Styles.bloco}
                onPress={() => props.navigation.navigate('Significado',{name:<b>NUARTE</b>, sig:'Núcleo de Arte'})}
      >
          <Text testID='btn17'>NUARTE</Text>
      </TouchableHighlight>

    </View>
  )
}


const Styles = StyleSheet.create({
  container:{
    flex: 1,
   
  },
  bloco:{
    width: 200,
    height: 40,
    backgroundColor: '#00ff40',
    padding: 10,
    justifyContent: 'center',
    alignItems: 'center',
    margin: 'auto',
    marginVertical: 10,
    borderRadius: 20,
  },

})