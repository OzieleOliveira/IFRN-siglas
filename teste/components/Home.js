import * as React from 'react';
import { Text, View, StyleSheet, TouchableHighlight } from 'react-native';


export default function Home(props){
  return (
    <View style={Styles.container}>

        <TouchableHighlight
        style={Styles.bloco}
                onPress={() => props.navigation.navigate('Significado',{name:<b>DG</b>, sig:'Direção Geral'})}
      >
          <Text testID='btnindex'><b>DG</b></Text>
      </TouchableHighlight>

    
      <TouchableHighlight
        style={Styles.bloco}
                onPress={() => props.navigation.navigate('Significado',{name:<b>GABIN</b>, sig:'Chefia de Gabinete'})}
      >
          <Text><b>GABIN</b></Text>
      </TouchableHighlight>

          <TouchableHighlight
        style={Styles.bloco}
                onPress={() => props.navigation.navigate('Significado',{name:<b>CTI</b>, sig:'Coordenação de Tecnologia da Informação'})}
      >
          <Text><b>CTI</b></Text>
      </TouchableHighlight>


      <TouchableHighlight
        style={Styles.bloco}
              onPress={() => props.navigation.navigate('Significado',{name:<b>COGPE</b>, sig:'Coordenação de Gestão de Pessoas'})}
      >
          <Text><b>COGPE</b></Text>         
      </TouchableHighlight>

          <TouchableHighlight
        style={Styles.bloco}
              onPress={() => props.navigation.navigate('Significado',{name:<b>COAPAC</b>,sig:'Coordenação de Apoio Acadêmico'})}
      >
          <Text><b>COAPAC</b></Text>
      </TouchableHighlight>

          <TouchableHighlight
        style={Styles.bloco}
              onPress={() => props.navigation.navigate('Significado',{name:<b>COAES</b>, sig:'Coordenação de Atividades Estudantis'})}
      >
          <Text><b>COAES</b></Text>
      </TouchableHighlight>

          <TouchableHighlight
        style={Styles.bloco}
              onPress={() => props.navigation.navigate('Significado',{name:<b>COEX</b>, sig:'Coordenação de Extensão'})}
      >
          <Text><b>COEX</b></Text>
      </TouchableHighlight>

          <TouchableHighlight
        style={Styles.bloco}
              onPress={() => props.navigation.navigate('Significado',{name:<b>COPEIN</b>, sig:'Coordenação de Pesquisa e Inovação'})}
      >
          <Text><b>COPEIN</b></Text>
      </TouchableHighlight>

          <TouchableHighlight
        style={Styles.bloco}
              onPress={() => props.navigation.navigate('Significado',{name:<b>DIAC</b>, sig:'Diretoria Acadêmica'})}
      >
          <Text><b>DIAC</b></Text>
      </TouchableHighlight>

          <TouchableHighlight
        style={Styles.bloco}
               onPress={() => props.navigation.navigate('Significado',{name:<b>SEAC</b>, sig:'Secretaria Acadêmica'})}
      >
          <Text><b>SEAC</b></Text>
      </TouchableHighlight>

          <TouchableHighlight
        style={Styles.bloco}
              onPress={() => props.navigation.navigate('Significado',{name:<b>COLAB</b>, sig:'Coordenação de Laboratórios'})}
      >
          <Text><b>COLAB</b></Text>
      </TouchableHighlight>

          <TouchableHighlight
        style={Styles.bloco}
              onPress={() => props.navigation.navigate('Significado',{name:<b>DIAD</b>, sig:'Diretoria de Administração'})}
      >
          <Text><b>DIAD</b></Text>
      </TouchableHighlight>

          <TouchableHighlight
        style={Styles.bloco}
              onPress={() => props.navigation.navigate('Significado',{name:<b>COSGEM</b>, sig:'Coordenação de Serviços Gerais e Manutenção'})}
      >
          <Text><b>COSGEM</b></Text>
      </TouchableHighlight>

          <TouchableHighlight
        style={Styles.bloco}
              onPress={() => props.navigation.navigate('Significado',{name:<b>COMPAT</b>, sig:'Coordenação de Material e Patrimônio'})}
      >
          <Text><b>COMPAT</b></Text>
      </TouchableHighlight>

          <TouchableHighlight
        style={Styles.bloco}
              onPress={() => props.navigation.navigate('Significado',{name:<b>COFINC</b>, sig:'Coordenação de Finanças e Contratos'})}
      >
          <Text><b>COFINC</b></Text>
      </TouchableHighlight>

          <TouchableHighlight
        style={Styles.bloco}
              onPress={() => props.navigation.navigate('Significado',{name:<b>NAPNE</b>, sig:'Núcleo de Atendimento às Pessoas com Necessidades Educacionais Especiais'})}
      >
          <Text><b>NAPNE</b></Text>
      </TouchableHighlight>

          <TouchableHighlight
        style={Styles.bloco}
              onPress={() => props.navigation.navigate('Significado',{name:<b>NEABI</b>, sig:'Núcleo de Estudos Afro-brasileiros e Indígenas do Campus Canguaretama'})}
      >
          <Text><b>NEABI</b></Text>
      </TouchableHighlight>

          <TouchableHighlight
        style={Styles.bloco}
                onPress={() => props.navigation.navigate('Significado',{name:<b>NUARTE</b>, sig:'Núcleo de Arte'})}
      >
          <Text><b>NUARTE</b></Text>
      </TouchableHighlight>

    </View>
  )
}


const Styles = StyleSheet.create({
  container:{
    flex: 1,
   
  },
  bloco:{
    width: 200,
    height: 40,
    backgroundColor: '#00ff40',
    padding: 10,
    justifyContent: 'center',
    alignItems: 'center',
    margin: 'auto',
    marginVertical: 10,
    borderRadius: 20,
  },

})