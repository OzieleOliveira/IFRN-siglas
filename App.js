import * as React from 'react';
import { Text, View, StyleSheet, TextInput, Button } from 'react-native';
import { NavigationContainer } from '@react-navigation/native';
import { createNativeStackNavigator } from '@react-navigation/native-stack'
import Home from './components/Home';
import Significado from './components/Significado';

const Stack = createNativeStackNavigator()

export default function App() {

  return (
      <NavigationContainer>
        <Stack.Navigator initialName='Home' >
        <Stack.Screen name='Siglas IFRN' component={Home} />
        <Stack.Screen name='Significado' component={Significado} />
        </Stack.Navigator>
      </NavigationContainer>
  );
}
  
 

