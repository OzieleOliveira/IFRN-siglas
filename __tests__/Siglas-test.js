import renderer from 'react-test-renderer'
import { render, fireEvent} from "@testing-library/react-native";
import Home from "../components/Home";
import App from "../App"
jest.mock("react-native/Libraries/Animated/NativeAnimatedHelper")

describe("teste de Snapshot", ()=>{
    it("Componentes de Home", ()=>{
        const home = renderer.create(<Home />).toJSON();
        expect(home).toMatchSnapshot();

    })
})

describe("teste de Inicialização e Exibição", ()=>{
    it("Componentes de Home", ()=>{
        const {getByText, getByTestId} = render(<Home/>)

        expect(getByText("DG")).toBeTruthy()
        expect(getByTestId("btn")).toBeTruthy();

        expect(getByText("GABIN")).toBeTruthy()
        expect(getByTestId("btn1")).toBeTruthy();

        expect(getByTestId("btn2")).toBeTruthy();
        expect(getByTestId("btn3")).toBeTruthy();
        expect(getByTestId("btn4")).toBeTruthy();
        expect(getByTestId("btn5")).toBeTruthy();
        expect(getByTestId("btn6")).toBeTruthy();
        expect(getByTestId("btn7")).toBeTruthy();
        expect(getByTestId("btn8")).toBeTruthy();
        expect(getByTestId("btn9")).toBeTruthy();
        expect(getByTestId("btn10")).toBeTruthy();
        expect(getByTestId("btn11")).toBeTruthy();
        expect(getByTestId("btn12")).toBeTruthy();
        expect(getByTestId("btn13")).toBeTruthy();
        expect(getByTestId("btn14")).toBeTruthy();
        expect(getByTestId("btn15")).toBeTruthy();
        expect(getByTestId("btn16")).toBeTruthy();
        expect(getByTestId("btn17")).toBeTruthy();
        
    })
})


describe("Testes de Navegação", ()=>{
    it ("Navegação de Index para Dobro",()=>{
        const{getByText, getByTestId} = render(<App />)

        const botao = getByTestId("bt")
        fireEvent.press(botao)

        const botao2 = getByTestId("bt2")
        fireEvent.press(botao2)
       
    })
})