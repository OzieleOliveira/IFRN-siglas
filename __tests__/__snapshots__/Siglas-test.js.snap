// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`teste de Snapshot Componentes de Home 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#00ff40",
        "borderRadius": 20,
        "height": 40,
        "justifyContent": "center",
        "margin": "auto",
        "marginVertical": 10,
        "padding": 10,
        "width": 200,
      }
    }
    testID="bt"
  >
    <Text
      testID="btn"
    >
      DG
    </Text>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#00ff40",
        "borderRadius": 20,
        "height": 40,
        "justifyContent": "center",
        "margin": "auto",
        "marginVertical": 10,
        "padding": 10,
        "width": 200,
      }
    }
    testID="bt2"
  >
    <Text
      testID="btn1"
    >
      GABIN
    </Text>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#00ff40",
        "borderRadius": 20,
        "height": 40,
        "justifyContent": "center",
        "margin": "auto",
        "marginVertical": 10,
        "padding": 10,
        "width": 200,
      }
    }
  >
    <Text
      testID="btn2"
    >
      CTI
    </Text>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#00ff40",
        "borderRadius": 20,
        "height": 40,
        "justifyContent": "center",
        "margin": "auto",
        "marginVertical": 10,
        "padding": 10,
        "width": 200,
      }
    }
  >
    <Text
      testID="btn3"
    >
      COGPE
    </Text>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#00ff40",
        "borderRadius": 20,
        "height": 40,
        "justifyContent": "center",
        "margin": "auto",
        "marginVertical": 10,
        "padding": 10,
        "width": 200,
      }
    }
  >
    <Text
      testID="btn4"
    >
      COAPAC
    </Text>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#00ff40",
        "borderRadius": 20,
        "height": 40,
        "justifyContent": "center",
        "margin": "auto",
        "marginVertical": 10,
        "padding": 10,
        "width": 200,
      }
    }
  >
    <Text
      testID="btn5"
    >
      COAES
    </Text>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#00ff40",
        "borderRadius": 20,
        "height": 40,
        "justifyContent": "center",
        "margin": "auto",
        "marginVertical": 10,
        "padding": 10,
        "width": 200,
      }
    }
  >
    <Text
      testID="btn6"
    >
      COEX
    </Text>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#00ff40",
        "borderRadius": 20,
        "height": 40,
        "justifyContent": "center",
        "margin": "auto",
        "marginVertical": 10,
        "padding": 10,
        "width": 200,
      }
    }
  >
    <Text
      testID="btn7"
    >
      COPEIN
    </Text>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#00ff40",
        "borderRadius": 20,
        "height": 40,
        "justifyContent": "center",
        "margin": "auto",
        "marginVertical": 10,
        "padding": 10,
        "width": 200,
      }
    }
  >
    <Text
      testID="btn8"
    >
      DIAC
    </Text>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#00ff40",
        "borderRadius": 20,
        "height": 40,
        "justifyContent": "center",
        "margin": "auto",
        "marginVertical": 10,
        "padding": 10,
        "width": 200,
      }
    }
  >
    <Text
      testID="btn9"
    >
      SEAC
    </Text>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#00ff40",
        "borderRadius": 20,
        "height": 40,
        "justifyContent": "center",
        "margin": "auto",
        "marginVertical": 10,
        "padding": 10,
        "width": 200,
      }
    }
  >
    <Text
      testID="btn10"
    >
      COLAB
    </Text>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#00ff40",
        "borderRadius": 20,
        "height": 40,
        "justifyContent": "center",
        "margin": "auto",
        "marginVertical": 10,
        "padding": 10,
        "width": 200,
      }
    }
  >
    <Text
      testID="btn11"
    >
      DIAD
    </Text>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#00ff40",
        "borderRadius": 20,
        "height": 40,
        "justifyContent": "center",
        "margin": "auto",
        "marginVertical": 10,
        "padding": 10,
        "width": 200,
      }
    }
  >
    <Text
      testID="btn12"
    >
      COSGEM
    </Text>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#00ff40",
        "borderRadius": 20,
        "height": 40,
        "justifyContent": "center",
        "margin": "auto",
        "marginVertical": 10,
        "padding": 10,
        "width": 200,
      }
    }
  >
    <Text
      testID="btn13"
    >
      COMPAT
    </Text>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#00ff40",
        "borderRadius": 20,
        "height": 40,
        "justifyContent": "center",
        "margin": "auto",
        "marginVertical": 10,
        "padding": 10,
        "width": 200,
      }
    }
  >
    <Text
      testID="btn14"
    >
      COFINC
    </Text>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#00ff40",
        "borderRadius": 20,
        "height": 40,
        "justifyContent": "center",
        "margin": "auto",
        "marginVertical": 10,
        "padding": 10,
        "width": 200,
      }
    }
  >
    <Text
      testID="btn15"
    >
      NAPNE
    </Text>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#00ff40",
        "borderRadius": 20,
        "height": 40,
        "justifyContent": "center",
        "margin": "auto",
        "marginVertical": 10,
        "padding": 10,
        "width": 200,
      }
    }
  >
    <Text
      testID="btn16"
    >
      NEABI
    </Text>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#00ff40",
        "borderRadius": 20,
        "height": 40,
        "justifyContent": "center",
        "margin": "auto",
        "marginVertical": 10,
        "padding": 10,
        "width": 200,
      }
    }
  >
    <Text
      testID="btn17"
    >
      NUARTE
    </Text>
  </View>
</View>
`;
